app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def young_to_old_actors
+    actors.order(:age)
+  end
+
+  def av_age_actors
```

Comment:
This is minor and nit-picky of me, but I would prefer the method name fully spell out what this method is doing. As an app grows and gets more complex, we don't want to have to spend mental energy trying to figure out what "av" means. Even though it's a little longer, I'd call this `average_age_actors` so there's no guessing.